src/UI/Actions.hs

```diff
@@ -458,6 +481,12 @@ instance Focusable 'ViewMail 'ManageMailTagsEditor where
 instance Focusable 'ViewMail 'ScrollingMailView where
   switchFocus _ _ = pure . set (asViews. vsViews . at ViewMail . _Just . vFocus) ScrollingMailView
 
+instance Focusable 'ViewMail 'ScrollingMailViewFindWordEditor where
+  switchFocus _ _ = pure
+                    . set (asViews. vsViews . at ViewMail . _Just . vFocus) ScrollingMailViewFindWordEditor
```

Comment:
Will need to reset/clear the input, otherwise it's confusing to have the last input still there.